.travis.yml

```diff
@@ -32,6 +32,9 @@ services:
 
 sudo: false
 
+before_install:
+  - composer config -g github-oauth.github.com $CI_USER_TOKEN
```

Comment:
It doesn't work,
https://travis-ci.org/Codeception/Codeception/jobs/98095911#L179

    Failed to download bower-asset/jquery.inputmask from dist: Could not authenticate against github.com
    Now trying to download from source

But I still benefit from composer cache when I am using --prefer-dist, right?